EventFilter/RPCRawToDigi/plugins/RPCAMCRawToDigi.cc

```diff
@@ -0,0 +1,378 @@
+#include "EventFilter/RPCRawToDigi/plugins/RPCAMCRawToDigi.h"
+
+#include <memory>
+
+#include "DataFormats/Common/interface/Handle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+#include "CondFormats/RPCObjects/interface/RPCAMCLink.h"
+#include "DataFormats/FEDRawData/interface/FEDHeader.h"
+#include "DataFormats/FEDRawData/interface/FEDRawData.h"
```

Comment:
"DataFormats/FEDRawData/interface/FEDRawDataCollection.h" already included in the header